autoconsent/autoconsent-impl/src/main/java/com/duckduckgo/autoconsent/impl/handlers/InitMessageHandlerPlugin.kt

```diff
@@ -69,14 +69,14 @@ class InitMessageHandlerPlugin @Inject constructor(
                     }
 
                     // Reset site
-                    autoconsentCallback.onResultReceived(consentManaged = false, optOutFailed = false, selfTestFailed = false)
+                    autoconsentCallback.onResultReceived(consentManaged = false, optOutFailed = false, selfTestFailed = false, isCosmetic = false)
 
                     val disabledCmps = repository.disabledCmps.map { it.name }
                     val autoAction = getAutoAction()
                     val enablePreHide = settingsRepository.userSetting
                     val detectRetries = 20
 
-                    val config = Config(enabled = true, autoAction, disabledCmps, enablePreHide, detectRetries)
+                    val config = Config(enabled = true, autoAction, disabledCmps, enablePreHide, detectRetries, enableCosmeticRules = true)
```

Comment:
Is this (enableCosmeticRules) something we plan to add to the remote config at some point?

Reply:
There's no such plans at the moment. We can still disable single rules by name, that should be enough.